### DISCUSSION TITLE
Regional map component

### DISCUSSION THREAD

**Discussion by wangzhenux** [twilio-labs/paste#2110] [Ideas] [answered]
Our team is working on a feature to onboard regional customers. It involves a global map to show where our customers are and what regions are available. I'd like to get some feedback on style and implementation. Also icons being used on that page if time available.
![image](https://user-images.githubusercontent.com/6403036/148078401-f79c2a74-e2c8-4613-8b36-2f741eba93ed.png)

**Comment by shleewhite** [comment, marked as answer]
@wangzhenux came to design system office hours to show us a prototype he built. The regional map component looks great and we had a few suggestions:
* Avoid using characters in icons because they cannot be internationalized easily
* Don't change the opacity of not currently selected locations on the map
* Make the land on the map slightly darker grey to increase color contrast against the white background
